pages/common/encfs.md

```diff
@@ -0,0 +1,21 @@
+# encfs
+
+> Encrypted virtual filesystem.
+> To unmount encrypted filesystem use `fusermount -u`.
```

Comment:
I´m not sure about this command description @bl-ue `Encrypted virtual filesystem` this tool is for manage or only mount?
about the `fusermount` I think we should create the `fusermount` page.

Reply:
@navarroaxel #5582

Reply:
I would recommend changing this to a `See also`. Perhaps something like this?
```suggestion
> Mounts or creates encrypted virtual filesystems.
> See also `fusermount`, which can unmount filesystems mounted by this command.
```

This also contains a suggestion for the top line too.